commons/all/src/test/java/org/infinispan/commons/util/TypedPropertiesTest.java

```diff
@@ -0,0 +1,73 @@
+package org.infinispan.commons.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Properties;
+
+import org.junit.Test;
+
+/**
+ * Test for {@link TypedProperties}.
+ *
+ * @author Diego Lovison
+ * @since 12.0
+ **/
+public class TypedPropertiesTest {
```

Comment:
it is missing the test with "incorrect" data, like doing  p.getIntProperty("boolean_put_str", 999) and check that 999 is returned.

Reply:
Done